.golangci.yaml

```diff
@@ -0,0 +1,146 @@
+run:
+  timeout: 5m
+  go: "1.19"
```

Comment:
```suggestion
  go: "1.20"
```
Since we updated it in go.mod

Reply:
Yeah, I wrote that part when CAAPH was on 1.19 and didn't update it later.